mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -81,7 +81,10 @@ def binary_cross_entropy(pred,
     """Calculate the binary CrossEntropy loss.
 
     Args:
-        pred (torch.Tensor): The prediction with shape (N, 1).
+        pred (torch.Tensor): The prediction with shape (N, 1) or (N, ).
+            When pred is of shape (N, 1) it will be expanded to one-hot
```

Comment:
what does `it will be expanded to one-hot format` mean? Does this mean pred's shape will be changed from what to what?